k-distribution/include/kframework/ktest-fail.mak

```diff
@@ -1,9 +1,10 @@
 # path to the current makefile
 MAKEFILE_PATH := $(dir $(abspath $(lastword $(MAKEFILE_LIST))))
 # path to the kompile binary of this distribuition
-KOMPILE=$(abspath $(MAKEFILE_PATH)/../../bin/kompile)
+K_BIN=$(abspath $(MAKEFILE_PATH)/../../bin)
+KOMPILE=${K_BIN}/kompile
 # and kdep
-KDEP=$(abspath $(MAKEFILE_PATH)/../../bin/kdep)
+KDEP=${K_BIN}/kdep
```

Comment:
using `K_BIN` here makes it much easier to run these tests in nix, since we only need to override the `K_BIN` env variable with the nix store path to the K binaries.